charts/node-server/templates/mongodb-cm.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-common-scripts
+  labels:
+    app: mongodb
+data:
+  readiness-probe.sh: |
+    #!/bin/bash
+    mongosh --eval 'db.hello().isWritablePrimary || db.hello().secondary' | grep -q 'true'
+  ping-mongodb.sh: |
+    #!/bin/bash
+    mongosh --eval 'db.adminCommand("ping")'
```

Add entrypoint script (see other comment)
```suggestion
    mongosh --eval 'db.adminCommand("ping")'
  entrypoint.sh: |
    #!/bin/bash
    mkdir -p /tl/data/db
    chmod 777 /tl/data/db
    exec mongod --dbpath /tl/data/db # could be cooler if it's an environment variable, so this path is just in the `deployment` yaml
```

---

note, that I know to run `mongod` because it is the [command in the docker file](https://hub.docker.com/layers/mongo/library/mongo/5.0.11/images/sha256-15342c724bd7c4be9b329078040bdd1f6629eca737d64fb0e249a08a914b31d8?context=explore) and `exec` makes the command "take over" the process, effectively terminating the `sh` process and replaces it with `mongod`, as the image author intended

---

so maybe call the script `mongodb-cmd.sh` instead?

---

Plus won't it be better to be in the `initContainers` config so we won't need to override the init cmd?

---

talked via f2f